zx.mjs

```diff
@@ -173,8 +175,11 @@ function transformMarkdown(source) {
 
 async function run(pieces, ...args) {
   let v = $.verbose
+  let q = $.quote
   $.verbose = false
+  $.quote = v => v
```

Comment:
Why this?

Reply:
`$.quote` adds redundant quotes on template render: `"`npm_config_yes=true npx tsc` --module ..."`

Reply:
Reverted.